### ISSUE TITLE
fxShow/fxHide breaks alignment

### ISSUE THREAD

**Issue by paistipoikka** [angular/flex-layout#184]
Hi,

When adding _fxShow/FxHide_ to my `<div>`, the _fxLayoutAlign_ alignment is no longer recognised. This can be easily replicated in the plunkr demo for this project. Fiddling around with various combinations make this easy to replicate.

- "@angular/flex-layout": "^2.0.0-beta.5"

If this is not a bug, would it be possible to provide an example? My understanding of the documentation is that I should be using `<fxFlexAlign>` for aligning the contents however that makes no difference either way. 

Many thanks

#### Works as intended:

```html
     <div fxLayout="row" fxFlex="grow" fxLayoutAlign="start center">
            <div fxFlex="200px" fxLayoutAlign="start center">
                Start Alignment
            </div>
            <!-- responsive menu -->
            <div fxFlex="grow" fxLayoutAlign="end center">
                End Alignment
            </div>
    </div>
``` 

#### Does **NOT** work:
```html
     <div fxLayout="row" fxFlex="grow" fxLayoutAlign="start center">
            <div fxFlex="200px" fxLayoutAlign="start center">
                Start Alignment
            </div>
            <!-- responsive menu -->
            <div fxFlex="grow" fxLayoutAlign="end center" fxShow fxHide.gt-sm>
                End Alignment
            </div>
    </div>
```

**Comment by i2egular** [comment]
In my case, i can fix by setting fxLayoutAlign outside and put another div inside then use fxShow or fxHide with it.

my example
```
<div fxLayout="row" fxLayoutAlign="center center">
    <div fxHide="false" fxHide.xs>This is center.</div>
</div>
```

**Comment by paistipoikka** [comment]
@i2egular thanks for that. Unfortunately, although this may work it creates a redundant div in the middle.

**Comment by ThomasBurleson** [comment]
Dual usages of fxShow + fxHide on the same element is not working properly. This has been fixed in PR #190. 

@paistipoikka - can you provide a Plunkr to validate that 190 fixes your issue ^?

**Comment by ThomasBurleson** [comment]
Fixed with SHA eee20b2
